tst/RtcpFunctionalityTest.cpp

```diff
@@ -175,6 +175,30 @@ TEST_F(RtcpFunctionalityTest, rembValueGet) {
     EXPECT_EQ(STATUS_RTCP_INPUT_REMB_INVALID, rembValueGet(invalidSSRCLength, SIZEOF(invalidSSRCLength), &maximumBitRate, ssrcList, &ssrcListLen));
 }
 
+static void on_PLI_callback(UINT64 opaque) {
+    *(PBOOL) opaque = TRUE;
+}
+
+TEST_F(RtcpFunctionalityTest, onpli) {
+    initializeEndianness();
```

Comment:
I think this should already be called for you, don't worry about calling in test

Reply:
ok